cpp/velox/substrait/SubstraitToVeloxPlan.cc

```diff
@@ -894,26 +894,6 @@ core::PlanNodePtr SubstraitToVeloxPlanConverter::toVeloxPlan(const ::substrait::
   }
 }
 
-bool isPushDownSupportedByFormat(
```

Comment:
Hi @zhztheplayer, this check seems not taking effect. Could you help confirm? Thanks!